src/components/ContextMenu/LiveHelperModal.js

```diff
@@ -21,14 +21,15 @@ const enhance = compose(
 );
 
 export const LiveHelperModal = ({
+  open,
   onHide,
   enabled,
   sec,
   // from recompose
   onEnabledClick,
   onSecChange
 }) => (
-  <Modal.Dialog>
+  <Modal show={open}>
```

Comment:
Why not use the same name `show`?

Reply:
Just felt weird to use a verb in props. Anyway, I'll rename it to `show` for all modals.

For states defined in the `ContextMenu`, how about changing them to `showingXXX` (openInputHelper -> showingInputHelper) ?

Reply:
For that, I prefer `showsInputHelper`, but I'm not against `showingXXX`. However, `openXXX` sounds like an action rather than a state.

Reply:
No problem. `showsInputHelper` works for me. I've renamed all of them with `show` on modal